images/entrypoint.sh

```diff
@@ -7,7 +7,7 @@ LOG_DIR=""
 LOG_LEVEL=10
 RESOURCE_PREFIX=""
 CONFIG_FILE=""
-CLI_PARAMS=""
+CLI_PARAMS="-v $LOG_LEVEL"
```

Comment:
why was this moved ?

this breaks setting rog level as cmdline parsing happens at L25